src/main/java/frc/robot/Robot.java

```diff
@@ -98,89 +98,18 @@ private void configureControls() {
     if (controlChooser.getSelected()==null){}
```

Comment:
I'd recommend using a [switch statement](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/switch.html) instead of a bunch of `else if` but that's more style than substance.